crates/forge/tests/e2e/common/runner.rs

```diff
@@ -51,6 +51,117 @@ pub(crate) fn setup_package(package_name: &str) -> TempDir {
     temp
 }
 
+pub(crate) fn setup_hello_workspace() -> TempDir {
```

Comment:
Why are we reconstructing whole scarb tomls here?

Reply:
I have to inject absolute path to `snforge_std` and I refuse to use toml_edit, it's unintuitive and has terrible API